test/tokenmanager.js

```diff
@@ -557,5 +563,49 @@ contract('Token Manager', ([root, holder, holder2, anyone]) => {
         await assertRevert(tokenManager.mint(anyone, 100))
       })
     })
+
+    context('toggle transferability', () => {
+
+      it('reverts when no permission', async() => {
+        await acl.revokePermission(ANY_ENTITY, tokenManager.address, TOGGLE_TRANSFERS_ROLE)
+        await assertRevert(tokenManager.enableTransfers(false), "APP_AUTH_FAILED")
+      })
+
+      it('changes transferability', async() => {
+        // We can't transfer when false
+        await tokenManager.enableTransfers(false)
+        assertRevert(token.transfer(holder2, 1, { from: holder }))
+
+        // We can transfer when true
+        await tokenManager.enableTransfers(true)
+        await token.approve(holder2, 10, { from: holder })
+        await token.transfer(holder2, 1, { from: holder })
+      })
+
+    })
+
+    context('set max tokens per account', () => {
+
+      it('reverts when no permission', async() => {
+        await acl.revokePermission(ANY_ENTITY, tokenManager.address, SET_MAX_TOKENS_ROLE)
+        await assertRevert(tokenManager.setMaxAccountTokens(100), "APP_AUTH_FAILED")
+      })
+
+      it('sets max amount of tokens', async() => {
+        await tokenManager.setMaxAccountTokens(100)
+        await assertRevert(tokenManager.mint(anyone, 101), 'TM_BALANCE_INC_NOT_ALLOWED')
+        await tokenManager.mint(holder, 99)
+        await tokenManager.mint(holder2, 99)
+        await token.approve(anyone, 99, { from: holder })
+        await token.transfer(anyone, 99, { from: holder })
+        await token.approve(anyone, 99, { from: holder2 })
+        await assertRevert(token.transfer(holder2, 99, { from: holder2 }))
+      })
+
+      it('uses 0 as infinity', async() => {
+        await tokenManager.setMaxAccountTokens(0)
+        await tokenManager.mint(anyone, 100000000)
```

Comment:
Perhaps you can use the max number to ensure it has set to it. `100000000` this isn't even 1 for a typical token with 18 decimals